README.md

```diff
@@ -40,7 +40,7 @@ See the section about [deployment](https://facebook.github.io/create-react-app/d
 
 ## Demo
 
-- Open [live demo](https://amal-covid-tracker.herokuapp.com/) 
+- Open [live demo](https://nchorage-03766.herokuapp.com/) 
```

Comment:
- [ ] Please update your live demo link, before sending any project that requires a Live demo link be sure it's deployed then submit.


![ooo](https://user-images.githubusercontent.com/61562626/142908751-288a3b14-12b7-4404-8635-18300d4c33d8.jpeg)

Reply:
This issue is well resolved! ✔️